### DISCUSSION TITLE
active directory account

### DISCUSSION THREAD

**Discussion by elenezet** [Security-Onion-Solutions/securityonion#1673] [Unsupported Versions] [answered]
Hello,
Is it possible to connect the SOC to the active directory to manage accounts centrally?

**Comment by TOoSmOotH** [comment, marked as answer]
https://docs.securityonion.net/en/2.3/faq.html?highlight=active#can-i-connect-security-onion-to-active-directory-or-ldap

**Reply by out-n-about** [reply]
I'm also looking for an answer on this.  I see that the guide says it doesn't recommend it, however can it be done, and how?  

Thanks

**Reply by TOoSmOotH** [reply]
Not at this time.
